cmake/FindcublasLt.cmake

```diff
@@ -0,0 +1,54 @@
+# ===============================================================================
+# Copyright 2020-2022 Intel Corporation Copyright 2020 Codeplay Software Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+# ===============================================================================
+
+find_package(CUDA 10.0 REQUIRED)
+
+# CUDA 12 adds support for int8 outputs for IMMA kernels
+if(CUDA_VERSION_MAJOR LESS 12)
+    # CUDA 12 required for int8 output 
+  add_definitions(-DDNNL_NO_IMMA_INT8_DST)
+endif()
```

Comment:
It seems to be used only in a single spot. why not just use CUDA_VERSION_MAJOR directly?